pypesto/petab/importer.py

```diff
@@ -304,6 +304,25 @@ def create_prior(self) -> NegLogParameterPriors:
         else:
             return None
 
+    def create_startpoint_method(self):
+        """
+        Creates a startpoint method, if the PEtab problem specifies an
+        initializationPrior. Returns None, if no initializationPrior
+        is specified.
+        """
+        if petab.INITIALIZATION_PRIOR_TYPE \
+                not in self.petab_problem.parameter_df:
+            return None
+        else:
+            def startpoint_method(n_samples: int, **kwargs):
```

Comment:
`kwargs` intentionally unused?

Reply:
yep. This was included, to make the interface consistent with the other start point methods, which have different key word arguments, depending on the method.

Reply:
(Which again is needed inside of `assign_startpoints`, that uses an unified interface for all those methods... as far as I understand)